.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+- If this is your first time, please read our contributor guidelines: https://git.k8s.io/community/contributors/guide#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
```

Comment:
The first contribution guide is somewhat applicable.. but the development-guide is not..
maybe replace that with: 
https://github.com/kubernetes-sigs/cri-tools/blob/master/CONTRIBUTING.md

Reply:
Yep, changed the development guide to te mentioned link.